contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
```

Comment:
> I believe we'll need to support an instance in one namespace being used by an app in another - so they can share data.

I agree that we will need to do this at some point (@bgrant0607 has said as much), but this is not going to happen anytime soon.  There are a lot of policy / auth issues around this that will need to be worked through.  I don't even want cross-ns links in MVP-1.